wordpress-mobile/gutenberg-mobile
# Fix pre block style to use monospace font.

Fixes #2081

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/21255

To test:
 - Start the demo app
 - Check the preformatted blocks are using a monospace font.

PR submission checklist:
